pycode/memilio-epidata/memilio/epidata_test/test_epidata_get_Case_Data.py

```diff
@@ -0,0 +1,1025 @@
+
+        file = 'cases_deaths.json'
+        f_read = os.path.join(directory, file)
+        df_deaths = pd.read_json(f_read)
+
+        data_list = df.columns.values.tolist()
+        self.assertEqual(
+            data_list, ["Date", "Confirmed", "Deaths", "Recovered"])
+        self.assertEqual(df[(df['Date'] == "2020-08-07")]['Confirmed'].item(),
+                         df_infected[(df_infected['Date'] == "2020-08-07")]['Confirmed'].item())
+        self.assertEqual(df[(df['Date'] == "2020-08-07")]
+                         ['Confirmed'].item(), 15)
+        self.assertEqual(
+            df[(df['Date'] == "2020-08-07")]['Deaths'].item(),
+            df_deaths[(df_deaths['Date'] == "2020-08-07")]['Deaths'].item())
+        self.assertEqual(df[(df['Date'] == "2020-08-07")]['Deaths'].item(), 2)
+        self.assertEqual(df[(df['Date'] == "2020-08-07")]
+                         ["Recovered"].item(), 14)
+        self.assertEqual(df[(df['Date'] == "2020-06-10")]
+                         ['Confirmed'].item(), 8)
+        self.assertEqual(df[(df['Date'] == "2020-06-10")]['Deaths'].item(), 1)
+        self.assertEqual(df[(df['Date'] == "2020-06-10")]
+                         ["Recovered"].item(), 8)
+
+        file = 'cases_all_age.json'
+        f_read = os.path.join(directory, file)
+        df_age = pd.read_json(f_read)
+        self.assertEqual(
+            df_age
+            [(df_age['Date'] == "2020-08-07") &
+             (df_age["Age_RKI"] == "A60-A79")]
+            ["Recovered"].item(),
+            6)
+        self.assertEqual(
+            df_age
+            [(df_age['Date'] == "2020-08-07") &
+             (df_age["Age_RKI"] == "A60-A79")]['Deaths'].item(),
+            2)
+        self.assertEqual(
+            df_age
+            [(df_age['Date'] == "2020-08-07") &
+             (df_age["Age_RKI"] == "A60-A79")]
+            ['Confirmed'].item(),
+            7)
+
+        file = 'cases_all_gender.json'
+        f_read = os.path.join(directory, file)
+        df_gender = pd.read_json(f_read)
+        self.assertEqual(
+            df_gender
+            [(df_gender['Date'] == "2020-08-07") &
+             (df_gender["Gender"] == "male")]["Recovered"].item(),
+            7)
+        self.assertEqual(
+            df_gender
+            [(df_gender['Date'] == "2020-08-07") &
+             (df_gender["Gender"] == "female")]['Deaths'].item(),
+            1)
+        self.assertEqual(
+            df_gender
+            [(df_gender['Date'] == "2020-08-07") &
+             (df_gender["Gender"] == "male")]['Deaths'].item(),
+            1)
+        self.assertEqual(
+            df_gender
+            [(df_gender['Date'] == "2020-08-07") &
+             (df_gender["Gender"] == "female")]['Confirmed'].item(),
+            7)
+
+        file = 'cases_all_county_gender.json'
+        f_read = os.path.join(directory, file)
+        df_gender = pd.read_json(f_read)
+        self.assertEqual(df_gender.shape[0], 17+411)
```

Comment:
I deleted one data set from the test data file which contained Eisenach since Eisennach is not anymore part of  the RKI data, thus instead of 412 rows we have only 411 rows. I'm looking into why also the 18 changed to 17.

Reply:
I now found why also the data for Berlin changed and here the 18 changed to 17. When editing the test sets to adapt them to the new format, I accidentially changed one date. I will correct this.